Dockerfile

```diff
@@ -2,9 +2,11 @@ FROM ubuntu:14.04
 MAINTAINER Kyle Anderson <kyle@xkyle.com>
 
 ENV DEBIAN_FRONTEND noninteractive
+RUN apt-get -y install software-properties-common
```

Comment:
why do we need this PPA at all? Is the version of firefox in ubuntu 14.04 not good enough?